### PR TITLE
Add forklift arm sprites

### PR DESCRIPTION
**Pull request by acepleiades**
#### Summary
Ultica "forklift arm sprites"

#### Content of the change
Adds forklift arm sprites

### CODE REVIEW

**Comment by acepleiades** [comment]
forklift bike
It works

File: gfx/UltimateCataclysm/pngs_normal_32x32/vehicle/forklift_arm/forklift_fork.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "id": [ "vp_forklift_fork" ],
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_normal_32x32/vehicle/forklift_arm/forklift_fork.json:3]:
```suggestion
    "id": "vp_forklift_fork",
```

Comment (acepleiades) [reply gfx/UltimateCataclysm/pngs_normal_32x32/vehicle/forklift_arm/forklift_fork.json:3]: It was like that before but it didn't work. I had to add the brackets in

Comment (I-am-Erk) [reply gfx/UltimateCataclysm/pngs_normal_32x32/vehicle/forklift_arm/forklift_fork.json:3]: That's super weird. You're sure no other changes?

Comment (int-ua) [reply gfx/UltimateCataclysm/pngs_normal_32x32/vehicle/forklift_arm/forklift_fork.json:3]: I'll check it locally and report back soon

Comment (int-ua) [reply gfx/UltimateCataclysm/pngs_normal_32x32/vehicle/forklift_arm/forklift_fork.json:3]: зображення
It does produce differing outputs but I don't expect it to affect the game. I'll have to download a fresh build though. Later.

Comment (int-ua) [reply gfx/UltimateCataclysm/pngs_normal_32x32/vehicle/forklift_arm/forklift_fork.json:3]: This is not something important though, you can ignore it and merge.

**Pull request merged**